src/services/user-service.ts

```diff
@@ -196,6 +199,80 @@ export const submitEvaluation = async (
       await EvaluationResultService.calculateScore(evaluation.evaluation_result_id)
     }
 
+    const filter = {
+      evaluation_result_id: evaluation.evaluation_result_id,
+      for_evaluation: true,
+      status: {
+        in: [EvaluationStatus.Open, EvaluationStatus.Ongoing, EvaluationStatus.ForRemoval],
+      },
+    }
+
+    if (user.is_external) {
+      Object.assign(filter, {
+        external_evaluator_id: user.id,
+      })
+    } else {
+      Object.assign(filter, {
+        evaluator_id: user.id,
+      })
+    }
+
+    const usesRemainingEvaluations = await EvaluationRepository.countAllByFilters(filter)
```

Comment:
typo? Should this be `userRemainingEvaluations`?

Reply:
😵‍💫